integration/flink/src/main/java/org/apache/carbon/core/metadata/StageManager.java

```diff
@@ -81,7 +81,7 @@ public static void writeStageInput(final String stageInputPath, final StageInput
   private static void writeSuccessFile(final String successFilePath) throws IOException {
     final DataOutputStream segmentStatusSuccessOutputStream =
         FileFactory.getDataOutputStream(successFilePath,
-            CarbonCommonConstants.BYTEBUFFER_SIZE, 1024);
+            CarbonCommonConstants.BYTEBUFFER_SIZE, 1024 * 1024 * 2);
```

Comment:
what if the file size is greater than 2 MB  ? why 2MB selected for this ?
may be need to pass the actual file size ?

Reply:
i set it 2M beacuase hdfs (dfs.namenode.fs-limits.min-block-size) configured minimum value size is 1M and in CarbonUtil.java class 
`getMaxOfBlockAndFileSize(long blockSize, long fileSize) `method use `long maxSize = blockSize;
    if (fileSize > blockSize) {
      maxSize = fileSize;
    }` 
if default size or filesize less than 1M program will get error ;
why 2M ? default is 1M so default * 2 bigger than it